openedx/features/edly/admin.py

```diff
@@ -0,0 +1,22 @@
+"""
+Django Admin pages for Edly.
+"""
+
+from django.contrib import admin
+
+from .models import EdlyOrganization, EdlySubOrganization
+
+
+class EdlyOrganizationAdmin(admin.ModelAdmin):
+    list_display = ['name', 'slug', 'created', 'modified']
+
+
+admin.site.register(EdlyOrganization, EdlyOrganizationAdmin)
+
+
+class EdlySubOrganizationAdmin(admin.ModelAdmin):
+    search_fields = ['lms_site__domain', 'lms_site__name', 'studio_site__domain', 'studio_site__name']
+    list_display = ['edx_organization', 'edly_organization', 'created', 'modified',]
```

Comment:
Admin still need to improve according to my comment.

Reply:
Done